app/api/orders.py

```diff
@@ -379,6 +381,13 @@ def before_update_object(self, order, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
+        if data.get('status') == 'pending':
```

Comment:
Can be completed or placed as well

Reply:
So should I just make this `data.get('status') != 'initializing'` ?

Reply:
No, because this should not run order is being cancelled.